src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -136,7 +135,8 @@ void tabulate() {
     ContestConfig config = ContestConfig.loadContestConfig(configPath);
     checkConfigVersionMatchesApp(config);
     boolean tabulationSuccess = false;
-    if (config.validate() && setUpLogging(config)) {
+    boolean isLoggingSetUp = setUpLogging(config);
+    if (config.validate() && isLoggingSetUp) {
```

Comment:
this is change in behavior. `setUpLogging` is now called with `confi` even if `config.validate()` is false. That doesn't seem right.

Reply:
Actually this is exactly the goal of the PR: we want the audit logs to now include failed validation messages.

Reply:
This API is pretty confusing. It reads to me like `config` may not be valid, yet we pass it as input to a function (which presumably expects it to be valid?). Does `setUpLogging` need the entire config object? Can we clarify that this is an acceptable usage?

Also, do we need a test?

Reply:
Good call, I made an attempt to clarify this more.

Re: the test, I don't think it's necessary -- all our tests so far revolve around the actual tabulation process, whereas this is just a slight change to what's included in the logs. I verified locally that the original issue identified in #566 is solved by this change.